# Robot state information
int32 id
float32[3] position
float32[3] orientation  # Roll, pitch, yaw
float32[] capabilities
float64 workload
bool failed